lib/manageiq/appliance_console/cli.rb

```diff
@@ -175,10 +187,19 @@ def parse(args)
         opt :oidc_unconfig,        "Unconfigure Appliance OpenID-Connect Authentication",            :type => :boolean, :default => false
         opt :server,               "{start|stop|restart} actions on evmserverd Server",   :type => :string
         opt :openscap,             "Setup OpenScap", :type => :boolean, :default => false
-        opt :messaging_hostname, "Messaging Hostname", :type => :string
-        opt :messaging_username, "Messaging Username", :type => :string
-        opt :messaging_password, "Messaging Password", :type => :string
-        opt :messaging_port,     "Messaging Port",     :type => :integer
+        opt :message_server_config,       "Configure Appliance as a Kafka Message Server",               :type => :boolean, :default => false
+        opt :message_server_unconfig,     "Unconfigure Appliance as a Kafka Message Server",             :type => :boolean, :default => false
+        opt :message_client_config,       "Configure Appliance as a Kafka Message Client",               :type => :boolean, :default => false
```

Comment:
Do we need a client config option?  If someone provides messaging server parameters it's assumed, right?

Reply:
I really liked the symmetry of having the four subcommands. It just seems like a clean user experience.

- message_server_config
- message_server_unconfig
- message_client_config
- message_client_unconfig

We wouldn't be able to use `message-server-host` to distinguish between a server or client config because `message-server-host` can be passed to the server config to request the server certs be configured with an IPAddress, which is `sadly` required for the IPAddress to work.

We could, however, user **--message-server-username** and  **--message-server-password** to distinguish a client config as those two parameters are required for the client config but not used by the server config.